trade-finance-manager-api/src/v1/helpers/generate-mandatory-criteria-string.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable no-restricted-syntax */
```

Comment:
not urgent - can you add a comment later about what this function does

Reply:
good shout